tests/v2/validation/nodescaling/replace.go

```diff
@@ -64,20 +64,25 @@ func MatchNodeToRole(t *testing.T, client *rancher.Client, clusterID string, isE
 func ReplaceNodes(t *testing.T, client *rancher.Client, clusterName string, isEtcd bool, isControlPlane bool, isWorker bool) {
 	clusterID, err := clusters.GetClusterIDByName(client, clusterName)
 	require.NoError(t, err)
-	numOfNodesBeforeDeletion, nodesToDelete := MatchNodeToRole(t, client, clusterID, isEtcd, isControlPlane, isWorker)
+	numOfNodesBeforeDeletion, nodeToDelete := MatchNodeToRole(t, client, clusterID, isEtcd, isControlPlane, isWorker)
```

Comment:
I can change it back, I don't really mind. I read it more as since we're deleting one node at a time, have it singular. We aren't deleting all the nodes at one time, so seemed weird to have it named like we did

Reply:
Resolved